csharp/src/Ice/Communicator.cs

```diff
@@ -44,13 +44,11 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
 
         /// <summary>Indicates whether or not object adapters created by this communicator accept non-secure incoming
         /// connections. When false, they accept only secure connections; when true, they accept both secure and
-        /// non-secure connections. This property corresponds to the Ice.AcceptNonSecureConnections configuration
+        /// non-secure connections. This property corresponds to the Ice.AcceptNonSecure configuration
         /// property. It can be overridden for each object adapter by the object adapter property with the same name.
         /// </summary>
-        // TODO: update doc with default value for AcceptNonSecureConnections - it's currently true but should be
-        // false.
-        // TODO: currently only this property is implemented and nothing else.
-        public bool AcceptNonSecureConnections { get; }
+        // TODO: update doc with default value for AcceptNonSecure - it's currently true but should be false
+        public bool AcceptNonSecure { get; }
```

Comment:
Any particular reason?

Reply:
Well I think is safer if you have to explicitly set this for each adapter, seems odd to have a setting to make default nonsecure.

Reply:
I could go either way I think. Any feelings @bernardnormier @bentoi ?

Reply:
It sounds reasonable but I guess this will make testing a little more complicated however since the driver won't be able to set `Ice.AcceptNonSecure` anymore and servers will have to set this on each OA.

Reply:
Currently I think it's more consistent to keep in Communicator as a default for OAs, because that's what we do for all other "inheritable" OA properties, namely IncomingFrameSizeMax (should be MaxSize), Locator, and Router.

If we made AcceptNonSecure OA-only, why not make IncomingFrameMaxSize OA-only too?
And Locator and Router (wouldn't be very inconvenient for Locator, and arguably what an OA needs is a LocatorRegistry, not a locator).